Journal of Aerospace Information Systems/Blockchain Properties for Near-Planetary, Interplanetary and Metaplanetary Space Domains/Blockchain Properties for Near-Planetary, Interplanetary and Metaplanetary Space Domains.tex

```diff
@@ -153,19 +153,19 @@ \section{Literature Review}\label{sec:litreview}
 \item as a means to coordinate the fulfillment of a desired operation carried out by many individual spacecraft\cite{mandl_bitcoin_2017}
 \end{itemize}
 
-Researchers with government, especially military, connections in China, Russia and the United States are actively investigating uses of blockchains for the access security and data integrity of Earth-orbiting satellites. Published Chinese military interest seems to be focused on preventing ``cyber \& physical attacks'' against space assets\cite{yu_blockchain_2019}, and to allow ``multiple departments to participate in the maintenance and update of equipment status''\cite{yu_situational_2019}. The Roscosmos State Corporation for Space Activities in Russia is developing a ``digital platform for control spacecrafts'' [sic] and for ``use of ground stations'' focused on the ``Roskosmos orbital group'' of satellites\cite{skobelev_towards_2018}. It has been suggested that ``the US military has taken a fancy to the anonymity of blockchain in recording transactions, and has begun to expand to the field of intelligence gathering to achieve covert targeted payments for incentive personnel.'' [sic]\cite{yu_situational_2019}. Mandl, at NASA Goddard Space Flight Center, has proposed using smart contracts on blockchains to create a ``Remote Sensing as a Service'' offering\cite{mandl_bitcoin_2017}. In Mandl's conception, a single Earth observation requirement could be obtained by multiple platforms conducting multiple observations under a variety of conditions until a desired goal is achieved.
+Researchers with government connections, especially military, in China, Russia and the United States are actively investigating uses of blockchains for access security and data integrity of Earth-orbiting satellites. Published Chinese military interest seems to be focused on preventing ``cyber \& physical attacks'' against space assets\cite{yu_blockchain_2019}, and to allow ``multiple departments to participate in the maintenance and update of equipment status''\cite{yu_situational_2019}. The Roscosmos State Corporation for Space Activities in Russia is developing a ``digital platform for control spacecrafts'' [sic] and for ``use of ground stations'' focused on the ``Roskosmos orbital group'' of satellites\cite{skobelev_towards_2018}. It has been suggested that ``the US military has taken a fancy to the anonymity of blockchain in recording transactions, and has begun to expand to the field of intelligence gathering to achieve covert targeted payments for incentive personnel.'' [sic]\cite{yu_situational_2019}. Mandl, at NASA Goddard Space Flight Center, has proposed using smart contracts on blockchains to create a ``Remote Sensing as a Service'' offering\cite{mandl_bitcoin_2017}. In Mandl's conception, a single Earth observation requirement could be obtained by multiple platforms conducting multiple observations under a variety of conditions until a desired goal is achieved.
```

Comment:
"Researchers with government connections, especially military," reads a bit oddly to me, although I don't object to the change.

How about "Researchers with government or military connections"?

Reply:
I think that is better. The initial version didn't read well, and I prefer your suggested change to mine